### ISSUE TITLE
Wiring images for airgapped installation is broken in 0.30.0

### ISSUE THREAD

**Issue by jasmdk** [getporter/porter#1355]
## Describe the bug
We are building airgapped bundles where images are packaged into the CNAB and the digest is then used when deploying applications, but this do not work in porter 0.30.0 - it just fails with missing digest parameter.

## To Reproduce
1. Run `porter create`
2. Add the following to porter.yaml:

images:
  alpine:
    imageType: docker
    repository: docker.io/alpine
    digest: "sha256:c0e9560cda118f9ec63ddefb4a173a2b2a0347082d7dff7dc14272e7841a5b5a"
    tag: latest

3. Reference the digest value - like an argument to install (as explained in https://porter.sh/author-bundles/#images):
install:
  - exec:
      description: "Install Hello World"
      command: ./helpers.sh
      arguments:
        - install
        - "{{ bundle.images.alpine.digest }}"
4. Run `porter install`
5. See error

## Expected behavior
In previous versions, the digest parameter of a referenced imaged could be used to specify a specific image in the CNAB bundle.
If you do not supply a digest parameter to the image, it will only work if you publish it to a registry and install it using --tag, but now neither works.

## Porter Command and Output
```
$ porter install
installing porter-hello...
WARNING: both registry and reference were provided; using the reference value of host.docker.internal:5443/temp/temp for the bundle reference
executing install action from porter-hello (installation: porter-hello)
Error: unable to resolve step: unable to render step template exec:
  arguments:
  - install
  - '{{ bundle.images.alpine.digest }}'
  command: ./helpers.sh
  description: Install Hello World
: Missing variable "digest"
Error: 1 error occurred:
        * container exit code: 1, message: <nil>
```

## Version
porter v0.30.0 (7a18716b)

**Comment by carolynvs** [comment]
This was an accidental regression from from #1308. We shouldn't have updated the manifest syntax, only changed how it is converted to a bundle.yaml 

https://github.com/getporter/porter/pull/1308/files#diff-c2df3d8f17e8cbab0d6ea3d15e4b1588d40d14f32b4066c4a144f7ae9c2e4dd9R479

We'll get that fixed and push a new release with the 2 fixes reported today as soon as possible. Thank you for the reports!

**Comment by carolynvs** [comment]
A fix is available in [v0.30.1](https://github.com/getporter/porter/releases/tag/v0.30.1)
